jaeger-core/src/main/java/com/uber/jaeger/propagation/ExceptionCatchingExtractorDecorator.java

```diff
@@ -0,0 +1,22 @@
+package com.uber.jaeger.propagation;
+
+import com.uber.jaeger.SpanContext;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RequiredArgsConstructor
```

Comment:
Could you add a javadoc, stating why this exists and how it works?

Reply:
you mean https://projectlombok.org/features/constructor?

Reply:
No, sorry, I meant a class-level javadoc for the classes `ExceptionCatching*Decorator`

Reply:
sure, I will do that although I think that  `Injector` and `Extractor` interfaces need more attention in this area.